easyrest/views/tag_controler.py

```diff
@@ -0,0 +1,21 @@
+from pyramid.view import view_config
+from pyramid.response import Response
+
+from pyramid.httpexceptions import HTTPNotFound
+
+from sqlalchemy.exc import DBAPIError
+
+from .. import models
+
+
+@view_config(route_name='get_tags', renderer='json', request_method='GET')
+def get_tags_controler(request):
+    try:
+        query = request.dbsession.query(models.Tag)
+        tags = query.all()
+        if len(tags) == 0:
+            raise HTTPNotFound()
+        dict_objs = [tag.as_dict() for tag in tags]
+    except DBAPIError:
+        return Response(db_err_msg, content_type='text/plain', status=500)
```

Comment:
I do not see where `db_err_msg` variable is defined. Anyway returning plain text with error message is not very good idea, as this output will be handled by JS on frontend side so make it more machine readable would be great. I'd propose for all JSON use the next format:
```
{
  "data": ... // result if no error
  "success": true/false // is request was succeeded
  "error": ... // error message if any
}
```

Reply:
It's something was not fixed from last review

Reply:
I`ve remove try, except block entirely.